client/src/ratings_reviews/ReviewsList.jsx

```diff
@@ -1,20 +1,16 @@
 import React from 'react';
+// import Rating from 'react-rating'; --> This is for later
+import ReviewTile from './ReviewTile.jsx';
 
-class ReviewsList extends React.Component {
```

Comment:
Please also change component to a functional component, you are getting the data as props, no need to have duplicate data as `state`